package/network/services/samba36/files/samba.init

```diff
@@ -109,10 +108,12 @@ start_service() {
 	procd_open_instance
 	procd_set_param command /usr/sbin/smbd -F
 	procd_set_param respawn
+	procd_set_param file /var/etc/smb.conf
 	procd_close_instance
```

Comment:
it seems this is also not related to the description of the patch. either split it into 2 patches or rework the subject/description